Windows.MSVC.toolchain.cmake

```diff
@@ -108,14 +109,16 @@ endif()
 
 # Find Visual Studio
 #
-findVisualStudio(
-    VERSION ${CMAKE_VS_VERSION_RANGE}
-    PRERELEASE ${CMAKE_VS_VERSION_PRERELEASE}
-    PRODUCTS ${CMAKE_VS_PRODUCTS}
-    PROPERTIES
-        installationVersion VS_INSTALLATION_VERSION
-        installationPath VS_INSTALLATION_PATH
-)
+if(NOT VS_INSTALLATION_PATH)
```

Comment:
With the Windows SDK's you're preferring eWDK environment variables - should this `VS_INSTALLATION_PATH` default to the `VSINSTALLDIR` environment variable if set?

Reply:
So I decided not to set a default for this from the environment as I didn't want to have situations where users would get unexpected VS versions being used. For example, consider a user that has multiple VS versions installed and tries to use this toolchain to select a specific one but they do so in a command prompt where VSINSTALLDIR is set. The expection would likely be that the version requested via the mechnism in the toolchain would be used rather than one set in VSINSTALLDIR.

What are your thoughts on this?

Reply:
With the idea to move the eWDK awareness to a separate toolchain file, keeping VSINSTALLDIR awareness out of `Windows.MSVC.toolchain.cmake` is the right thing to do.